src/env-dev.json

```diff
@@ -4,7 +4,7 @@
     "http": "http://127.0.0.1:8545",
     "ws": "ws://127.0.0.1:8546"
   },
-  "network-id": 4,
+  "network-id": 12346,
```

Comment:
@JohnDanz Was this intentional?

Reply:
yeah, why are we pointing to rinkeby for local dev when we have a docker node running clique? tired of changing it on my branches, I don't believe there is any reason this has to be 4 (rinkeby).I'm trying to confirm with @stephensprinkle but he is DND today. Pretty sure this is fine, if you don't want to include it, that's fine, it's more for convenience

Reply:
I've mentioned this before, but it bears repeating: there's no reason to specify a network ID here unless it's important from a UX point of view to double-check that the ID (manually entered into this config file) matches the one the ethereum node is using.  Otherwise, the Ethereum node has only 1 network ID associated with it, so it's not an extra piece of info you have to specify.

My suggestion is just to get rid of this setting altogether, and just use network that the Ethereum node to which you're connected is on.

Reply:
It has def gone back and forth. It would be awesome if we could drop it altogether.

Reply:
Update: we're going to get rid of this setting in [ch6692].  Until then, don't worry about what this is set to, because it's going to be different depending on each dev environment.  (Personally, I usually dev against a local Rinkeby node, so I use 4, but I know others use 12346, etc.)